src/components/UserPage.js

```diff
@@ -22,32 +36,32 @@ const UserPage = props => {
     console.log(username);
   }
   //this if statement will greet a user with a list of their videos, otherwise it will send a prompt to create a video
-  if (getAllUserVideos) {
-    return (
-      <Container>
-        <div>
-          <h1>{greeting}</h1>
-          <h2>Videos</h2>
-          {props.userVideos.map(video => {
-            return (
-              <div Key={video.id}>
-                <h3>{video.video_title}</h3>
-                <div>{video.location}</div>
-              </div>
-            );
-          })}
-        </div>
-      </Container>
-    );
-  }
+
   return (
     <>
       <Container>
         <h1>{greeting}</h1>
-        <p>
-          Looks like you haven't created any videos yet, lets {""}
-          <Link to="/create">create some videos!</Link>
-        </p>
+        <div style={videoListContainer}>
+          <div style={VideoThumbsContainer}>
+            {props.userVideos && props.userVideos ? (
+              props.userVideos.map(video => {
+                return (
+                  <div Key={video.id}>
+                    <h3>{video.video_title}</h3>
+                    <Link to={`/videos/${video.id}`}>
+                      <Thumbnail video={video} />
```

Comment:
We may want to consider another solution other than Thumbnail, or understand the differences between Thumbnail and Video